src/odinapi/utils/encrypt_util.py

```diff
@@ -10,18 +11,24 @@
 
 
 def encrypt(msg):
-    msg = msg + ' '*(16 - (len(msg) % 16 or 16))
-    cipher = AES.new(SECRET_KEY.encode(), AES.MODE_ECB)
-    return base64.urlsafe_b64encode(
-        cipher.encrypt(msg.encode())
-    ).decode()
+    cipher = AES.new(base64.b64decode(SECRET_KEY.encode()), AES.MODE_EAX)
```

Comment:
Why is the key base64 decoded?

Reply:
because get_random_bytes returns our key as a bytestring that can not be decoded to unicode without the base64 stuff. And we store our key in an ini-file, so there is some encoding stuff needed.

Reply:
alright, that makes sense, I suppose the decoded bytes are 16 long then and not the encoded b64 string.